src/main/scala/wdl4s/expression/FileEvaluator.scala

```diff
@@ -115,7 +115,7 @@ case class FileEvaluator(valueEvaluator: ValueEvaluator, coerceTo: WdlType = Wdl
       case a: Ast if a.isArrayOrMapLookup =>
         evalValue(a) match {
           case Success(f: WdlFile) => Success(Seq(f))
-          case _ => evaluateRecursive(a.getAttribute("rhs"))
+          case _ => evaluateRecursive(a.getAttribute("lhs"))
```

Comment:
Also @kshakir I'm buying what you're selling. But I reserve the right to say that I sided with @mcovarr all along if travis winds up irritated!